notebooks/official/feature_store/sdk-feature-store.ipynb

```diff
@@ -40,7 +40,7 @@
         "      <img src=\"https://cloud.google.com/ml-engine/images/github-logo-32px.png\" alt=\"GitHub logo\">\n",
```

Comment:
<p>There might be something wrong with the Vertex AI workbench link.</p><p> </p><p>Add H1 heading in same cell, but above links</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/913/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>